### PR TITLE
@mzikherman - Add suggested artists schema for me

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: schema/me/suggested_artists.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import {
+  GraphQLBoolean,
+  GraphQLList,
+  GraphQLString,
+  GraphQLInt,
+} from 'graphql';
+
+export default {
+  type: new GraphQLList(Artist.type),
+  description: 'A list of the current user’s suggested artist, based on a single artist',
```
Comment (mzikherman) [inline schema/me/suggested_artists.js]: suggested artist**s**

**Comment by mzikherman** [comment]
Tight, bro.

File: schema/me/suggested_artists.js (hunk 1 above)
Comment (broskoski) [inline schema/me/suggested_artists.js]: HOLD THIS PR!

**Comment by broskoski** [comment]
Thx, bro.

**Pull request merged**